arithmetization/src/main/java/net/consensys/linea/zktracer/types/Utils.java

```diff
@@ -83,4 +88,15 @@ public static BitDecOutput bitDecomposition(int input, int nbStep) {
     }
     return output;
   }
+
+  public static Map<String, Integer> computeSpillings() throws IOException {
+    final Map<String, Integer> spillings = new ConcurrentHashMap<>();
```

Comment:
does this map need to be modified? because otherwise if it is only read, then it could be a simple immutable one